package.json

```diff
@@ -34,7 +34,7 @@
     "react-native-modal-selector-searchable": "^2.1.4",
     "react-native-paper": "4.7.2",
     "react-native-paper-dropdown": "^1.0.7",
-    "react-native-reanimated": "1.13.3",
+    "react-native-reanimated": "^2.2.4",
```

Comment:
@kumaransoftsuave were from you've to install 2.2.4 versions? I think the latest stable version is 2.3.0
![image](https://user-images.githubusercontent.com/35322132/145387234-4bba1731-74f0-43e6-bff9-89f181f4b82c.png)

Reply:
2.2.4 is ok

Reply:
Great